src/memtable.rs

```diff
@@ -615,13 +683,13 @@ impl MemTable {
     #[cfg(test)]
     fn consistency_check(&self) {
         let mut seen_append = false;
-        let mut last_index = None;
+        // let mut last_index = None;
         for idx in self.entry_indexes.iter() {
             // Check 1: indexes are contiguous.
-            if let Some(last_index) = last_index {
-                assert_eq!(idx.index, last_index + 1);
-            }
-            last_index = Some(idx.index);
+            // if let Some(last_index) = last_index {
```

Comment:
why comment out?

Reply:
No need for this check because entry index doesn't store log index anymore.